python/llm/test/benchmark/csv_to_html.py

```diff
@@ -39,18 +39,54 @@ def main():
     if len(csv_files)>1:
         data2 = pd.read_csv(csv_files[1], index_col=0)
 
+        data1_df=pd.DataFrame(data1)
+        data2_df=pd.DataFrame(data2)
```

Comment:
do we need this? they are already dataframes, no?

Reply:
Yes, will remove this step.